server/model/status_page.js

```diff
@@ -46,7 +47,14 @@ class StatusPage extends BeanModel {
      */
     static async renderHTML(indexHTML, statusPage) {
         const $ = cheerio.load(indexHTML);
-        const description155 = statusPage.description?.substring(0, 155) ?? "";
+
+        let description155 = "";
+        if (statusPage.description) {
```

Comment:
Have not looked into this (am currently In the train on my way to work) what happens if there is no such if clause? 🤔